utils/build-presets.ini

```diff
@@ -2328,10 +2328,11 @@ no-assertions
 [preset: mixin_codesigning]
 darwin-toolchain-application-cert=lldb_codesign
 
-# Note: tensorflow_osx_base does not mix in "buildbot_osx_package" or
-# "mixin_osx_package_base" because they enable ios/tvos/watchos and tests by
-# default.
-# Instead, options are copied here individually.
+# Note: `tensorflow_osx_base` does not mix in `buildbot_osx_package` because
+# they enable ios/tvos/watchos by default. Instead, options are copied here
+# individually.
+# TODO(TF-941): Try to mixin `buildbot_osx_package` directly. This requires
```

Comment:
Note: I'm looking into TF-941 now. If `tensorflow_osx_base` can mixin `buildbot_osx_package` directly, we won't encounter this kind of error in the future.